content/en/network_monitoring/performance/network_page.md

```diff
@@ -33,7 +33,11 @@ The next example shows all aggregate connections from IP addresses representing
 
 You can set the timeframe over which traffic is aggregated using the time selector at the top right of the page:
 
-{{< img src="network_performance_monitoring/network_page/npm_timeframe.png" alt="Time frame NPM"  style="width:30%;">}} 
+{{< img src="network_performance_monitoring/network_page/npm_timeframe.png" alt="Time frame NPM"  style="width:30%;">}}
+
+Tags from Datadog integrations are unified service tagging can be used for aggregating  and filtering automatically. See
+[custom facets]({{< ref "#custom-facets" >}}) below for other tags. You can also select "Popular Tags" to aggregate using on a combination of
+`service`, `kube_service`, `short_image`, and `container_name`.
```

Comment:
```suggestion
Tags from Datadog integrations or Unified Service Tagging can be used for aggregating and filtering automatically. See [custom facets](#custom-facets), below, for other tags. You can also select "Popular Tags" to aggregate on, using a combination of `service`, `kube_service`, `short_image`, and `container_name`.
```

not sure if there's a reason you weren't using the native markdown syntax for anchor links

Reply:
Only ignorance!